code/__DEFINES/dcs/signals/signals_turf.dm

```diff
@@ -4,3 +4,5 @@
 
 ///from base of atom/has_gravity(): (atom/asker, list/forced_gravities)
 #define COMSIG_TURF_HAS_GRAVITY "turf_has_gravity"
+
+#define COMSIG_TURF_ENTERED "turf_entered"
```

Comment:
Same thing, this isn't needed, we have `COMSIG_ATOM_ENTERED` that applies to all atoms, turfs included